agr_java_core/src/main/java/org/alliancegenome/api/entity/AGMPhenotypeAnnotationDocument.java

```diff
@@ -0,0 +1,22 @@
+package org.alliancegenome.api.entity;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonView;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.alliancegenome.curation_api.model.entities.AffectedGenomicModel;
+import org.alliancegenome.neo4j.view.View;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@JsonPropertyOrder({"subject", "gene", "relation", "object", "primaryAnnotations"})
+@JsonView({View.DiseaseAnnotationAll.class})
+public class AGMPhenotypeAnnotationDocument extends PhenotypeAnnotationDocument {
+
+	private AffectedGenomicModel subject;
+
+	public AGMPhenotypeAnnotationDocument() {
+		setCategory("agm_phenotype_annotation");
```

Comment:
Here the text is hardcoded, but in the other two its done via a static variable. Please be consistant.